index.bs

```diff
@@ -173,9 +173,6 @@ to set the [=navigation params/attribution source=] of |navigationParams| to |at
 
 <h4 id="monkeypatch-document-creation">Document creation</h4>
 
-At the time <a spec="html">create and initialize a <code>Document</code> object</a> is invoked, the user agent knows the final URL used for the
```

Comment:
I think most likely all of this routing through navigate steps can be ripped out, but given the attributionsrc= side isn't implemented in the draft spec, I think this state is fine for now.

Reply:
Yeah, I agree.